sample/src/main/AndroidManifest.xml

```diff
@@ -12,6 +13,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/Theme.AppCompat.Light.DarkActionBar"
+        tools:ignore="GoogleAppIndexingWarning"
```

Comment:
It simply removes a Lint warning and highlight in Android Studio complaining about missing app indexing. I assume you won't ever need deep link support through Google search for this sample app. ;)